pkg/inits/config.go

```diff
@@ -27,7 +27,7 @@ func GetConfig() *entities.Config {
 	once.Do(func() {
 		viper.SetConfigName("config")
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("../../")
+		viper.AddConfigPath("/root")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/7#issuecomment-2264507398>#758118</a><span> - 08/02/2024, 04:16 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Changing the configuration path to a hardcoded "/root" directory can cause issues in different environments where the application might not have access to the "/root" directory. This can lead to the application failing to start due to the inability to read the configuration file.
</div>
<div id="fix">
<b>Fix:</b> Add multiple configuration paths to ensure the configuration file is found in different environments. This will make the application more flexible and robust.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -30 +30 @@
 - viper.AddConfigPath("/root")
 + viper.AddConfigPath("/root")
 + viper.AddConfigPath(".")
 + viper.AddConfigPath("/etc/productivite")
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
Will resolve in future